### PR TITLE
RDA-1785 : Ingestion of Story data type

### PR DESCRIPTION
**Pull request by Rodriguez-Carlos-BAH**
- Refactoring formatters by adding FormatterFactory and FormatterInterface
- Fix Update for existing documents
- Add dhType property
- Update Unit test

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,4 +6,4 @@ __pycache__/
 .scannerwork/
 .env
 deploy.sh
-.venv*
```
Comment (nickpricedev) [inline .gitignore:9]: Can we leave .venv* as well? I use it.

**Review by nickpricedev — CHANGES_REQUESTED** [review]
Quick change to the .gitignore as noted. Everything else looks good

File: .gitignore (hunk 1 above)
Comment (Rodriguez-Carlos-BAH) [reply .gitignore:9]: ok

**Review by nickpricedev — APPROVED** [review]
Looks good. Approved

**Pull request merged**